lightning/src/ln/channelmanager.rs

```diff
@@ -4490,6 +4374,72 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		self.our_network_pubkey.clone()
 	}
 
+	/// Handles a channel reentering a functional state, either due to reconnect or a monitor
+	/// update completion.
+	fn handle_channel_resumption(&self, pending_msg_events: &mut Vec<MessageSendEvent>,
+		channel: &mut Channel<<K::Target as KeysInterface>::Signer>, raa: Option<msgs::RevokeAndACK>,
+		commitment_update: Option<msgs::CommitmentUpdate>, order: RAACommitmentOrder,
+		pending_forwards: Vec<(PendingHTLCInfo, u64)>, funding_broadcastable: Option<Transaction>,
+		channel_ready: Option<msgs::ChannelReady>, announcement_sigs: Option<msgs::AnnouncementSignatures>)
+	-> (Option<(u64, OutPoint, Vec<(PendingHTLCInfo, u64)>)>, Result<(), MsgHandleErrInternal>) {
```

Comment:
I don't think this needs to return a `Result` anymore, I don't think it'll ever `Err` anymore